src/OrchardCore/OrchardCore.Data.YesSql.Abstractions/DbConnectionValidatorResult.cs

```diff
@@ -2,21 +2,24 @@ namespace OrchardCore.Data;
 
 public enum DbConnectionValidatorResult
 {
-    // Unknown indicates that the connection string status is unknown or was not yet validated
+    // 'Unknown' indicates that the connection string status is unknown or was not yet validated.
     Unknown,
 
-    // NoProvider indicated that the provider is missing
+    // 'NoProvider' indicated that the provider is missing.
     NoProvider,
 
-    // DocumentNotFound indicates that the connection string was valid, yet the Document table does not exist 
-    DocumentNotFound,
+    // 'DocumentTableNotFound' indicates that the connection string was valid, yet the Document table does not exist.
+    DocumentTableNotFound,
 
-    // DocumentFound indicates that the connection string was valid, yet the Document table exist
-    DocumentFound,
+    // 'DocumentFound' indicates that the connection string was valid, yet the Document table exist.
+    DocumentTableFound,
 
-    // InvalidConnection unable to open a connection to the given connection string
+    // 'ShellDescriptorDocumentFound' indicates that the connection string was valid and the document table exists with at least one entry of shell-descriptor document.
+    ShellDescriptorDocumentFound,
+
+    // 'InvalidConnection' unable to open a connection to the given connection string.
     InvalidConnection,
 
-    // UnsupportedProvider indicated invalid or unsupported database provider
+    // 'UnsupportedProvider' indicated invalid or unsupported database provider.
```

Comment:
If you use comments, use the `<summary>` tag, also just before the `DbConnectionValidatorResult` declaration.

    /// <summary>
    /// Some comments.
    /// </summary>

Reply:
@jtkech done.